src/components/entries/FEEL/Feel.js

```diff
@@ -194,7 +195,8 @@ function FeelTextfield(props) {
       tooltipContainer,
       type,
       value: feelOnlyValue,
-      variables
+      variables,
+      feelLanguageContext,
```

Comment:
Consider: Is this specific to FEEL? If so, then keep the name, if not, then consider to change to `languageContext`. The popup can open with different types, and other languages may in the future also provide dialects, built-ins, etc.

Reply:
Good. Let's keep this like that for the moment.